htdocs/js/jquery.fileupload.js

```diff
@@ -209,10 +213,13 @@ $(function() {
         $.extend( image, data );
         $field.data( "image-attributes", image );
 
+        var escape_titletext = image.title.replace( /</g, '&lt;' ).replace( /'/g, "&apos;" );
```

Comment:
It boggles my mind that there isn't a function for this, but I did peek around and didn't see one... just a bunch of people on the Internet saying 'do it yourself'. And I guess we don't have such a function already anywhere?